autoload/vundle/scripts.vim

```diff
@@ -254,4 +254,36 @@ func! s:load_scripts(bang)
   return eval(readfile(f, 'b')[0])
 endf
 
+
+" ---------------------------------------------------------------------------
+" Get the dependencies of the given bundle.
+"
+" bundle -- The bundle to check. (must be installed)
+" return -- List of bundle names.
+" ---------------------------------------------------------------------------
+func! vundle#scripts#getdeps(bundle)
+  if filereadable(a:bundle['rtpath'] . '/depends.txt')
+    return eval("['" . join(readfile(a:bundle['rtpath'] . '/depends.txt'), "','") . "']")
```

Comment:
You know, I suggested the removal of this particular item, but in any case, let me note that you start with an array of lines, then join it with commas and quotes, and then eval it, so you end up with... an array of lines.